### ISSUE TITLE
fyne-cross err : invalid for option uid

### ISSUE THREAD

**Issue by yunxi177** [lucor/fyne-cross#6]
```
PS D:\store\examples> fyne-cross --targets=linux/amd64,windows/amd64,darwin/amd64 github.com/fyne-io/examples
Downloading dependencies
Value "S-1-5-21-378678101-2309852114-219419761-500" invalid for option uid (number expected)
adduser [--home DIR] [--shell SHELL] [--no-create-home] [--uid ID]
[--firstuid ID] [--lastuid ID] [--gecos GECOS] [--ingroup GROUP | --gid ID]
[--disabled-password] [--disabled-login] [--add_extra_groups] USER
   Add a normal user

adduser --system [--home DIR] [--shell SHELL] [--no-create-home] [--uid ID]
[--gecos GECOS] [--group | --ingroup GROUP | --gid ID] [--disabled-password]
[--disabled-login] [--add_extra_groups] USER
   Add a system user

adduser --group [--gid ID] GROUP
addgroup [--gid ID] GROUP
   Add a user group

addgroup --system [--gid ID] GROUP
   Add a system group

adduser USER GROUP
   Add an existing user to an existing group

general options:
  --quiet | -q      don't give process information to stdout
  --force-badname   allow usernames which do not match the
                    NAME_REGEX configuration variable
  --help | -h       usage message
  --version | -v    version number and copyright
  --conf | -c FILE  use FILE as configuration file

exit status 1
```

**Comment by lucor** [comment]
@yunxi177 it should be fixed in  50a19ef.
Could you please update to the latest version and let me know if it works ?

**Comment by yunxi177** [comment]
It works
